### PR TITLE
fix: update CODEOWNERS

### PR DESCRIPTION
**Pull request by mxdvl**
## What does this change?

Update CODEOWNERS with a valid list.

## Why?

@oliverlloyd can’t review all PRs! 😥

### CODE REVIEW

**Review by bryophyta — APPROVED** [review]
I'm not sure that reviews are actually restricted to code owners at the moment (I copied branch protection rules from DCR which doesn't seem to have that requirement either?)

But this seems like a good change 👍

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -1 +1 @@
-*       @gtrufitt @oliverlloyd @liywjl
+*       @guardian/dotcom-platform @oliverlloyd
```
Comment (oliverlloyd) [inline .github/CODEOWNERS:1]:
```suggestion
*       @guardian/dotcom-platform
```

Comment (oliverlloyd) [reply .github/CODEOWNERS]: You can remove me completely, I don't mind 😄

**Review by oliverlloyd — APPROVED** [review]
Thanks!

**Comment by mxdvl** [comment]
This is only so reviews are requested automatically.

**Pull request merged**